js/views/roomlistview.js

```diff
@@ -272,6 +265,14 @@
 				this.renameRoom(newRoomName);
 			}
 		},
+		renameKeyUp: function(e) {
+			if (e.keyCode === 13) {
+				this.confirmRoomRename();
+			} else if (e.keyCode === 27) {
+				this.$el.find('.rename-element').addClass('hidden-important');
```

Comment:
lgtm with a small change: can you move these two lines in a separate function like `showRenameInput`? Call from here as `showRenameInput(false)` to hide the input and show the button and adjust the other places that do the same?